src/main/java/hudson/plugins/tfs/model/PingCommand.java

```diff
@@ -0,0 +1,61 @@
+package hudson.plugins.tfs.model;
+
+import hudson.model.AbstractProject;
+import jenkins.util.TimeDuration;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.StaplerRequest;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class PingCommand extends AbstractCommand {
+
+    public static class Factory implements AbstractCommand.Factory {
+        @Override
+        public AbstractCommand create() {
+            return new PingCommand();
+        }
+
+        @Override
+        public String getSampleRequestPayload() {
+            return "{\n" +
+                    "    \"parameter\":\n" +
+                    "    [\n" +
+                    "        {\"name\":\"id\",\"value\":\"123\"},\n" +
+                    "        {\"name\":\"verbosity\",\"value\":\"high\"}\n" +
+                    "    ],\n" +
+                    "    \"team-parameters\":\n" +

```

Comment:
Shouldn't it be "team-parameter" (i.e. singular) to be consistent with the "parameter" item?